images/linux/scripts/installers/powershellcore.sh

```diff
@@ -6,10 +6,3 @@
 
 # Install Powershell
 apt-get install -y powershell
-
-# Run tests to determine that the software installed as expected
-echo "Testing to make sure that script performed as expected, and basic scenarios work"
-if ! command -v pwsh; then
```

Comment:
Shouldn't we do `invoke_tests "Tools" "PowerShell Core"` here?

Reply:
@AlenaSviridenko we have decided to remove this test, because we use `powershellcore` in testing.

Reply:
ok, but probably then we should remove test suite for PowerShell Core https://github.com/actions/virtual-environments/pull/2374/files#diff-566d9fbecb5003c69d8bf0a9de1f159b931a82e63c7d96acd1ab48c0f40c768cR86

because I don't see where it is called. At least in current `main`

Reply:
Thank you! Decided to bring back `pwsh` test.